politeiawww/piwww.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/decred/politeia/plugins/comments"
 	piplugin "github.com/decred/politeia/plugins/pi"
+	ticketvote "github.com/decred/politeia/plugins/ticketvote"
```

Comment:
What's the purpose of aliasing this?  If you're refering to a package by its proper name, you don't need to alias it, unless its a versioned api package like `v1`.

Reply:
somehow automatically happened